src/core/public/index.ts

```diff
@@ -66,28 +73,24 @@ export { CoreContext, CoreSystem } from './core_system';
  * https://github.com/Microsoft/web-build-tools/issues/1237
  */
 export interface CoreSetup {
-  /** {@link ApplicationSetup} */
-  application: ApplicationSetup;
-  /** {@link I18nSetup} */
-  i18n: I18nSetup;
-  /** {@link InjectedMetadataSetup} */
-  injectedMetadata: InjectedMetadataSetup;
```

Comment:
Since I've changed the type of `ui/platform` to be `InternalCoreSetup` for now I think this is fine. I want it to be clear that any plugins depending on this  functionality won't be able to in the new platform.

The two uses I've seen so far aren't necessary in NP, though I'm not 100% sure of other injected var usages.